src/operator/tensor/elemwise_binary_scalar_op_extended.cc

```diff
@@ -49,8 +49,38 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_minimum_scalar)
     .set_attr_parser(ParamParser<NumpyBinaryScalarParam>)
     .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Backward<cpu, mshadow_op::le>);
 
+#if MXNET_USE_ONEDNN == 1
+bool PowerStorageType(const nnvm::NodeAttrs& attrs,
+                      const int dev_mask,
+                      DispatchMode* dispatch_mode,
+                      std::vector<int>* inputs,
+                      std::vector<int>* outputs) {
+  CHECK_EQ(inputs->size(), 1);
+  CHECK_EQ(outputs->size(), 1);
+
+  return DNNLStorageType(attrs, dev_mask, true, dispatch_mode, inputs, outputs);
+}
+
+void PowerComputeExCPU(const nnvm::NodeAttrs& attrs,
+                       const OpContext& ctx,
+                       const std::vector<mxnet::NDArray>& inputs,
+                       const std::vector<OpReqType>& req,
+                       const std::vector<mxnet::NDArray>& outputs) {
+  if (SupportDNNL(inputs[0])) {
```

Comment:
This function allows only ndim equal to 1, 2 and 4. Are you sure this function should be so strict?